index.d.ts

```diff
@@ -138,6 +146,8 @@ export type CommonOptions<EncodingType extends EncodingOption = DefaultEncodingO
 	- file URL.
 	- web [`WritableStream`](https://developer.mozilla.org/en-US/docs/Web/API/WritableStream).
 
+	This can be an array of values such as `['inherit', 'pipe']` or `[filePath, 'pipe']`.
+
```

Comment:
Maybe also link to the tip here: https://github.com/sindresorhus/execa/pull/643/files#diff-5a831ea67cf5cf8703b0de46901ab25bd191f56b320053be9332d9a3b0d01d15R812

Reply:
Fixed. :+1: